src/GafferUI/StandardGraphLayout.cpp

```diff
@@ -715,7 +715,7 @@ class LayoutEngine
 			Direction idealDirection;
 
 			// True if edge is representing an auxiliary connection
-			bool auxiliary;
+			bool auxiliary = false;
```

Comment:
I'm surprised this is necessary if the same bool on line 703 can remain uninitialized. You sure you don't need to patch up both?

Reply:
@andrewkaufman oddly it didn't need it there and compiled fine, I can add it in anyways though...

Reply:
As far as I can see the auxiliary flag on the Edges is not even used at the moment. I might have added that before changing the implementation a bit. Maybe I'm missing something, but it looked to me like we're only ever accessing that struct member for Vertices. I could look into whether it's ok to just remove it.